vendors/nordic/boards/nrf52840-dk/aws_tests/config_files/iot_flash_config.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * FreeRTOS
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * http://aws.amazon.com/freertos
+ * http://www.FreeRTOS.org
+ */
+
+/**
+ * @file   iot_flash_config.h
+ */
+
+#ifndef _AWS_COMMON_IO_FLASH_CONFIG_H_
+#define _AWS_COMMON_IO_FLASH_CONFIG_H_
+
+/* Set defaults which are not overridden */
+#include "iot_flash_config_defaults.h"
```

Comment:
I don't like header files that simply include 1 other header file.

Reply:
Added a logging macro so it's not emtpy. This `*_config.h` pattern is common among various modules. Some modules have hard dependencies on such config files, however this flash module does not at the moment.

There's a number of potential enhancements that could be made to our flash layout across all boards, and when that time comes, this stub may become more useful.

Currently, it primarily serves as a stub and to reinforce our `*_config.h` pattern